Source/JavaScriptCore/yarr/YarrInterpreter.cpp

```diff
@@ -115,9 +115,9 @@ class Interpreter {
 
         static size_t allocationSize(unsigned numberOfFrames)
         {
-            static_assert(alignof(DisjunctionContext) <= sizeof(void*));
+            static_assert(alignof(DisjunctionContext) <= alignof(max_align_t));
             size_t rawSize = sizeof(DisjunctionContext) - sizeof(uintptr_t) + Checked<size_t>(numberOfFrames) * sizeof(uintptr_t);
-            size_t roundedSize = roundUpToMultipleOf<sizeof(void*)>(rawSize);
+            size_t roundedSize = roundUpToMultipleOf<alignof(max_align_t)>(rawSize);
```

Comment:
It sounds wrong in 64bit environment as it becomes 16

Reply:
My interpretation of this static assert was that we were only making sure that we didn't break the alignment guarantees of this class. Is there some other part of the static_assert that I am missing?